### PR TITLE
Bump version to v5.5.6

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
New ABI version: 10.1.6

### CODE REVIEW

File: rpm/spec.in [hunk 1]
```diff
@@ -43,7 +43,7 @@ you will need to install %{name}-devel.
 mkdir -p $RPM_BUILD_ROOT/
 
 %check
-%{__make} check
+#%{__make} check
```
Comment (gautam-kotian-sociomantic) [inline rpm/spec.in]: :skeptical-andrei:

Comment (andreas-bok-sociomantic) [reply rpm/spec.in]: shit,

**Comment by andreas-bok-sociomantic** [comment]
Made Andrei happy

**Comment by andreas-bok-sociomantic** [comment]
Feel free to merge if there are no further comments :)

**Review by gautam-kotian-sociomantic — APPROVED** [review]
LGTM, but I don't have merge rights.

**Comment by andreas-bok-sociomantic** [comment]
Thanks

**Pull request merged**